standard-integration/server/node/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "type": "module",
   "dependencies": {
-    "@paypal/paypal-server-sdk": "^0.5.1",
+    "@paypal/paypal-server-sdk": "^0.6.0",
```

Comment:
For node the intent enum has changed from `CheckoutPaymentIntent.CAPTURE` to `CheckoutPaymentIntent.Capture`